src/ui/components/HeroSection/index.js

```diff
@@ -10,7 +10,10 @@ import './styles.scss';
 
 type Props = {|
   children?: any,
+  // eslint-disable-next-line react/no-unused-prop-types
```

Comment:
that's why I think we should not have that

Reply:
hmm ok I'll see if I can the title another way..